### PR TITLE
updatePhoneNumberToContact reducer and test

### PR DESCRIPTION
**Pull request by radhikamattoo**

### CODE REVIEW

File: src/stores/defaults.ts [hunk 1]
```diff
@@ -52,7 +52,13 @@ export const storeStateDefault: StoreState = {
   ],
   currentMatchIndex: 1,
   matchIdToMatchState: {},
-  phoneNumberToContact: {},
+  phoneNumberToContact: {
+    phoneNumber: {
+      phoneNumber: '0987654321',
```
Comment (yoav-zibin) [inline src/stores/defaults.ts]: Change:

    phoneNumber: {
       phoneNumber: '0987654321',

to:

    '+1987654321': {
       phoneNumber: '+1987654321',

File: src/reducers/index.test.ts [hunk 2]
```diff
@@ -107,4 +109,33 @@ it('setCurrentMatchIndex', () => {
   };
   expect(reduce(initialState, action)).toEqual(expectedState);
 });
+
+it('updatePhoneNumberToContact', () => {
+  const initialState = storeStateDefault;
+  let { phoneNumberToContact, ...rest } = initialState;
+
+  let someContact: Contact = {
+    phoneNumber: '1234567890',
```
Comment (yoav-zibin) [inline src/reducers/index.test.ts]: phoneNumber: '1234567890',

to

     phoneNumber: '+1234567890',

File: src/reducers/index.test.ts [hunk 3]
```diff
@@ -107,4 +109,33 @@ it('setCurrentMatchIndex', () => {
...
+    name: 'someName',
+    avatarImage: 'someImage'
+  };
+
+  let newPhoneNumberToContact: PhoneNumberToContact = {};
+  newPhoneNumberToContact['phoneNumber'] = someContact;
```
Comment (yoav-zibin) [inline src/reducers/index.test.ts]: newPhoneNumberToContact['phoneNumber'] = someContact;
to
newPhoneNumberToContact['+1234567890'] = someContact;

**Comment by radhikamattoo** [comment]
@yoav-zibin Done!

**Pull request merged**